docs/assets/javascript/elements/uxdot-pattern.css

```diff
@@ -1,89 +1,96 @@
+@import url('/styles/rh-code-block-lightdom.css');
+
 :host {
+  container: host / inline-size;
   display: block;
-  container-name: host;
-  container-type: inline-size;
-  margin-block-end: var(--rh-space-2xl, 32px);
+  margin-block-start: var(--rh-space-2xl);
 }
 
 #container {
   display: grid;
-  grid-template-columns: 1fr;
-  grid-template-areas: 'controls' 'example' 'code';
-  gap: var(--rh-space-2xl, 32px) var(--rh-space-4xl, 64px);
-  padding: var(--rh-space-2xl, 32px);
-  border: var(--rh-border-width-sm, 1px) solid var(--rh-color-border-subtle);
-  border-radius: var(--rh-border-radius-default, 3px);
-  background-color: var(--rh-color-surface);
-  color: var(--rh-color-text-primary);
-  border-color: var(--rh-color-border-subtle);
+  grid-template-columns: auto auto;
+  grid-template-areas: 'heading' 'controls' 'content' 'code';
+  gap: var(--rh-space-2xl) var(--rh-space-4xl);
 
-  --rh-color-surface: inherit;
-}
+  @container host (width >= 530px) {
+    grid-template-areas:
+      'heading controls'
+      'content content'
+      'code code';
+  }
 
-#container h3,
-#container ::slotted(h2),
-#container ::slotted(h3),
-#container ::slotted(h4),
-#container ::slotted(h5),
-#container ::slotted(h6) {
-  margin-block: 0 var(--rh-space-lg, 16px) !important;
-  font-size: var(--rh-font-size-body-text-lg, 1.125rem) !important;
+  @container host (width >= 992px) {
+    grid-template-areas:
+      'heading heading'
+      'controls controls'
+      'example example'
+      'code code';
+  }
 }
 
-#color-picker {
-  grid-area: controls;
-  justify-self: flex-end;
+#heading {
+  grid-area: heading;
   display: flex;
   align-items: center;
-  gap: var(--rh-space-lg, 16px);
 }
 
-#example {
-  grid-area: example;
-}
+#description { grid-area: content; }
```

Comment:
likely this was meant to be:
```Suggestion
#description { 
  grid-area: description; 
}
```

Reply:
might also need a `display: none` on it unless something is slotted and the grid is setup to handle the display somewhere which currently isn't.  but the duplicate content grid-areas were also conflicting